static/app/components/smartSearchBar/index.tsx

```diff
@@ -1073,15 +1084,23 @@ class SmartSearchBar extends Component<Props, State> {
     if (searchTerm) {
       const preparedSearchTerm = prepareQuery ? prepareQuery(searchTerm) : searchTerm;
 
-      tagKeys = filterKeysFromQuery(tagKeys, preparedSearchTerm);
+      tagKeys = filterKeysFromQuery(
+        tagKeys,
+        preparedSearchTerm,
+        this.props.fieldDefinitionGetter
```

Comment:
nit, can we destructure `fieldDefinitionGetter`

Reply:
yeah, done!